### DISCUSSION TITLE
Ctrl+I box stuck at "Generating". Inline suggestions still working.

### DISCUSSION THREAD

**Discussion by threaddy** [community/community#118500] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

After updating Copilot VScode extension, the `Ctrl + I ` box stopped working. After sending the prompt, the box is stuck at "Generating". 
OS: Windows 11. Copilot extension v1.178.0. VScode 1.88.00.
![Screenshot 2024-04-10 083458](https://github.com/community/community/assets/29302845/164408d0-5e76-43ee-ba8b-e06bef47c2a6)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by jfullstackdev** [comment]
it's working fine for me 

![image](https://github.com/community/community/assets/47092464/0b22078d-6a64-4af0-8ce0-4a80d64d9906)
